### ISSUE TITLE
Add support for crop

### ISSUE THREAD

**Issue by jlfwong** [jlfwong/speedscope#39]

**Comment by sandyarmstrong** [comment]
Absolutely, would love this! Sometimes I have a trace with some big noisy event that has a lot of frames that are also around beforehand. I'd like to only look at the trace from _before_ the big noisy event so I can better analyze those frames in context.

**Comment by jlfwong** [comment]
#366 contains a version of this already implemented. I'll take a closer look at it before next release -- I think it doesn't handle multithreaded profiles well yet, and since this is a destructive operation, it would be nice to provide the ability to undo this operation without reloading the whole profile. This would be the first need for undo in the system, since this is the first destructive operation.
